lib/modules/manager/pixi/readme.md

```diff
@@ -1,5 +1,13 @@
-Limited support for pixi package manager
+### Pixi package manager
 
-Whenever the pixi config in `pyproject.toml` or `pixi.toml` file is updated, the `pixi.lock` file will be checked for updates as well.
+Currently support channels from anaconda and prefix.dev, other channels are not supported yet.
 
-Currently only lock file maintenance is supported.
+Whenever the pixi config in `pyproject.toml` or `pixi.toml` file is updated, `pixi.lock` file will be checked for updates as well.
+
+Renovatebot will pick pixi based on known lock file version and pixi version mapping to avoid upgrade the version of lock file,
+optionally you can set `constraints.pixi` in your config to override this.
```

Comment:
```suggestion
Renovatebot will pick a pixi version based on known mapping between lockfile version and pixi version.
This is done in order to avoid upgrading the lock file version.
Optionally you can set `constraints.pixi` in your config to override this.
```